frontend/src/components/maps/leaflet/MapProperties.test.tsx

```diff
@@ -3,85 +3,129 @@ import renderer from 'react-test-renderer';
 import { ParcelPopupView } from 'components/maps/ParcelPopupView';
 import { IParcelDetail, IParcel } from 'actions/parcelsActions';
 import Map from './Map';
-import { Marker } from 'react-leaflet'
-import {mount, shallow} from 'enzyme'
+import { Marker } from 'react-leaflet';
+import { mount, shallow } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
-import Enzyme from 'enzyme'
+import Enzyme from 'enzyme';
 import { render } from '@testing-library/react';
 
 Enzyme.configure({ adapter: new Adapter() });
- 
+
 // This will spoof the active parcel (the one that will populate the popup details)
-const mockDetails = {
-    id: 1,
-    pid: '000-000-000',
-    latitude: 48,
-    longitude: 123,
-    propertyStatus: 'active',
-    propertyClassification: 'Core Operational',
-    description: 'test',
-    assessedValue: 1000000,
-    address: {
-        line1: '1234 mock Street',
-        line2: 'N/A',
-        city: 'Victoria',
-        province: 'BC',
-        postal: 'V1V1V1',
-    },
-    landArea: 'unknown',
-    landLegalDescription: 'test',
-    buildings: [],
-    agency: 'FIN',
-} as IParcelDetail
+const mockDetails: IParcelDetail = {
+  id: 1,
+  pid: '000-000-000',
+  latitude: 48,
+  longitude: 123,
+  propertyStatus: 'active',
+  classification: 'Core Operational',
+  description: 'test',
+  assessedValue: 1000000,
+  address: {
+    line1: '1234 mock Street',
+    line2: 'N/A',
+    city: 'Victoria',
+    province: 'BC',
+    postal: 'V1V1V1',
+  },
+  landArea: 'unknown',
+  landLegalDescription: 'test',
+  buildings: [],
+  agency: 'FIN',
+};
 
-// To check for alert message 
+// To check for alert message
 const emptyDetails = null;
 
-
-// This mocks the parcels of land a user can see - should be able to see 2 markers 
-const mockParcels = [{id: 1, latitude: 48, longitude: 123}, {id: 2, latitude: 50, longitude: 133}] as IParcel[]
+// This mocks the parcels of land a user can see - should be able to see 2 markers
+const mockParcels = [
+  { id: 1, latitude: 48, longitude: 123 },
+  { id: 2, latitude: 50, longitude: 133 },
+] as IParcel[];
 const noParcels = [] as IParcel[];
 
-// Check that the markers have correct po sition given the mock parcel 1 above
-it('Renders the marker in correct position', ()=> {
-    const component = mount(<Map lat={48.43} lng={-123.37} zoom={14} parcels={mockParcels} activeParcel={mockDetails} onParcelClick={jest.fn()} />);
-    const marker = component.find(Marker).first();
-    expect(marker.prop('position')).toStrictEqual([48,123]);
+// Check that the markers have correct position given the mock parcel 1 above
+it('Renders the marker in correct position', () => {
+  const component = mount(
+    <Map
+      lat={48.43}
+      lng={-123.37}
+      zoom={14}
+      parcels={mockParcels}
+      activeParcel={mockDetails}
+      agencies={[]}
+      propertyClassifications={[]}
```

Comment:
I had to add **agencies** and **propertyClassifications** to the unit tests to be able to compile